### DISCUSSION TITLE
Is it mandatory to have node >=14.17 to use semantic-release

### DISCUSSION THREAD

**Discussion by abhayJava** [semantic-release/semantic-release#2280] [Q&A]
Is it mandatory to have node >=14.17 to use semantic-release? How to use semantic-release if my node is v12.22 , as its not possible to update to 14.17

**Comment by travi** [comment]
most modern CI services enable use of different node versions between stages. we recommend using whatever your project needs for tasks like build, verification, etc but switch to a more recent version for release (ideally the latest LTS version).

if for some reason this is not possible, older versions of semantic-release have been quite stable, so you could use an older version that allows use of an older node version (although we don't recommend this option if the first suggestion is possible for you)

**Reply by abhayJava** [reply]
i have node 14.15.4 version. Which would be minimum  compatible version of semantic-release. Also , how do i use npx semantic-release command to use specific version of semantic-release

**Reply by travi** [reply]
> how do i use npx semantic-release command to use specific version of semantic-release

https://docs.npmjs.com/cli/v8/commands/npx

> i have node 14.15.4 version. Which would be minimum compatible version of semantic-release.

that version does not satisfy the minimum required by the current semantic-release version

**Reply by abhayJava** [reply]
i understand it doesnt satisfy current version , how about older versions of semantic-release , which older version of semnatic-release is compatible with node 14.15.4

**Reply by afreidz** [reply]
just curious, what is it about 14.17 that makes it a requirement over 14.16.x? i didnt see anything immediately clear from the node changelog.
